src/types/function.ts

```diff
@@ -15,6 +16,7 @@ export interface IFunctionPaths {
 export interface IFunctionValues {
   original: any;
   given: any;
+  resolved?: Resolved;
```

Comment:
@P0lip Hmmm. This `?` modifier makes me a little sick in my mouth. However, removing makes TS cringe because some tests (eg. `alphabetical.test.ts`) need a fake `IFunctionValues` instance... and I haven't found an easy to build a fake `Resolved`.

I might come back to this later, though... because well, it doesn't make sense to make this property optional (especially for such dummy reasons).

Reply:
> I might come back to this later, though... because well, it doesn't make sense to make this property optional (especially for such dummy reasons).

Yeah, you are right. It'd be indeed ideal if we didn't need to make this optional, since it's not optional in reality.

I can help you with sorting out tests if you don't mind.

In general, if a function does not make use of Resolved, you could use `any` type assertion, i..e `{ original: [], given: [] } as any`. Not perfect, but should do the trick for now.

Reply:
Fixed